### PR TITLE
Last suzuka stable with faucet patch fix

### PR DESCRIPTION
**Pull request by 0xmovses**
## Description
Applies the patch fix from to the last stable commit of this fork. 

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: crates/aptos-faucet/core/src/server/run.rs [hunk 1]
```diff
@@ -276,7 +276,9 @@ impl RunConfig {
             handler_config: HandlerConfig {
                 use_helpful_errors: true,
                 return_rejections_early: false,
-                max_concurrent_requests: None,
+                // Replace None to Some(1) to force to send one Tx at a time.
+                // Patch to avoid sequence number mix issue.
+                max_concurrent_requests: Some(1),
```
Comment (mzabaluev) [inline crates/aptos-faucet/core/src/server/run.rs:281]: OK

File: aptos-move/vm-genesis/src/lib.rs [hunk 2]
```diff
@@ -417,12 +417,12 @@ fn initialize(
...
-    let rewards_rate_numerator = genesis_config.rewards_apy_percentage
-        * (rewards_rate_denominator / 100)
-        * genesis_config.epoch_duration_secs
-        / NUM_SECONDS_PER_YEAR;
+    let _num_epochs_in_a_year = NUM_SECONDS_PER_YEAR / genesis_config.epoch_duration_secs;
+    // Multiplication before division to minimize rounding errors due to integer division.
+    /*let rewards_rate_numerator = (genesis_config.rewards_apy_percentage * rewards_rate_denominator
+        / 100)
+        / num_epochs_in_a_year;*/
+    let rewards_rate_numerator = 0; // todo: feature flag this for Movement Labs
```
Comment (mzabaluev) [inline aptos-move/vm-genesis/src/lib.rs:425]: I would prefer commenting out the previous revision rather than replacing it with commented-out upstream code (which is equivalent but buggy), since I have actually fixed the division by zero bug there once.

**Review by mzabaluev — APPROVED** [review]

**Comment by 0xmovses** [comment]
Testing first in `movement`, will merge after confirmation of success.

**Comment by mzabaluev** [comment]
Needs to be updated against the base branch (which earlier was the branch of the merged #40 for whatever reason).

**Comment by mzabaluev** [comment]
The base branch is misleading here. It is not based on #30 and would need to be extensively re-tested if merged on top of that. `last-suzuka-stable` branches off the last known good commit in `monza` and adds the patches for the faucet issue and the reward rate.

File: aptos-move/vm-genesis/src/lib.rs (hunk 2 above)
Comment (mzabaluev) [reply aptos-move/vm-genesis/src/lib.rs]: This was an artifact of the base choice, gone now.

**Comment by 0xmovses** [comment]
Yes, well noticed. So I suppose another manual test run against #30 one that is in review state.
